app/commands/git/sync_concept_exercise.rb

```diff
@@ -27,8 +27,8 @@ def call
         prerequisites: find_concepts(exercise_config[:prerequisites])
       )
 
-      SyncAuthors.(exercise)
-      SyncContributors.(exercise)
+      SyncExerciseAuthors.(exercise)
+      SyncExerciseContributors.(exercise)
```

Comment:
Now that we have classes to sync concepts too, I had to rename the existing classes.